### PR TITLE
Replace "link" with "url"

### PR DESCRIPTION
**Pull request by 2Cubed**
Also, add `github` option to `!cactus`.

### CODE REVIEW

File: cactusbot/commands/magic/cactus.py [hunk 1]
```diff
@@ -22,19 +22,47 @@ class Cactus(Command):
...
-            ("link", "https://twitter.com/CactusDevTeam",
+            ("url", "https://twitter.com/CactusDevTeam",
              "twitter.com/CactusDevTeam")
         )
 
     @Command.command()
+    async def github(self, project=None):
+        if project is None or project.lower() in ("bot", "cactusbot"):
+            return MessagePacket(
+                "Check out my GitHub repository at: ",
```
Comment (Innectic) [inline cactusbot/commands/magic/cactus.py]: I don't think we should include this.

File: cactusbot/commands/magic/cactus.py [hunk 2]
```diff
@@ -22,19 +22,47 @@ class Cactus(Command):
...
+                ("url", "https://github.com/CactusDev/CactusBot",
+                 "github.com/CactusDev/CactusBot")
+            )
+        elif project.lower() == "issue":
+            return MessagePacket(
+                "Create a GitHub issue at: ",
```
Comment (Innectic) [inline cactusbot/commands/magic/cactus.py]: Or this

File: cactusbot/commands/magic/cactus.py [hunk 3]
```diff
@@ -22,19 +22,47 @@ class Cactus(Command):
...
+                ("url", "https://github.com/CactusDev/CactusBot/issues",
+                 "github.com/CactusDev/CactusBot/issues")
+            )
+        elif project.lower() in ("api", "cactusapi"):
+            return MessagePacket(
+                "Check out the GitHub repository for CactusAPI at: ",
```
Comment (Innectic) [inline cactusbot/commands/magic/cactus.py]: Or this

File: cactusbot/commands/magic/cactus.py [hunk 4]
```diff
@@ -22,19 +22,47 @@ class Cactus(Command):
...
+                ("url", "https://github.com/CactusDev/CactusAPI",
+                 "github.com/CactusDev/CactusAPI")
+            )
+        elif project.lower() == "sepal":
```
Comment (Innectic) [inline cactusbot/commands/magic/cactus.py]: Or this

File: cactusbot/commands/magic/cactus.py (hunk 1 above)
Comment (2Cubed) [inline cactusbot/commands/magic/cactus.py]: @Innectic The entire command, or just the text preceding the URL?

**Pull request merged**